lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.h

```diff
@@ -325,15 +323,46 @@ class SymbolFileDWARF : public SymbolFileCommon {
     m_file_index = file_index;
   }
 
-protected:
   typedef llvm::DenseMap<const DWARFDebugInfoEntry *, Type *> DIEToTypePtr;
-  typedef llvm::DenseMap<const DWARFDebugInfoEntry *, lldb::VariableSP>
-      DIEToVariableSP;
+
+  virtual DIEToTypePtr &GetDIEToType() { return m_die_to_type; }
+
   typedef llvm::DenseMap<const DWARFDebugInfoEntry *,
                          lldb::opaque_compiler_type_t>
       DIEToClangType;
+
+  virtual DIEToClangType &GetForwardDeclDieToClangType() {
```

Comment:
`DIEToClangType` could be renamed to `DieToCompilerType`
`ClangTypeToDIE` could be renamed to `ComilerTypeToDIE`
Anything that is a map of something to `lldb::opaque_compiler_type_t`, or `lldb::opaque_compiler_type_t` to something can be renamed to be "CompilerType" instead of "ClangType"

Reply:
Awesome! I'll do that